dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -0,0 +1,497 @@
+﻿using System;
+using System.Windows.Input;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.Foundation;
+using InfoBar_TestUI;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace MUXControlsTestApp
```

Comment:
>MUXControlsTestApp [](start = 10, length = 18)

I'm a bit confused, why is this file being added? I'm looking at the repro and it looks like it already exists. Did something weird happen with the git history?

Reply:
Yeah, you're right. An issue arose because this PR contains the old commits from the initial PR. I merged the latest in master to this branch and pushed it again, so hopefully it fixed the problem.